docs/commands/csv2po.rst

```diff
@@ -14,16 +13,18 @@ your PO translations.
 If you are starting out with your own CSV files (not created by po2csv), take
 note of the assumptions of the column layout explained below.
 
+
 .. _csv2po#usage:
 
 Usage
 =====
 
-::
+.. code-block:: console
```

Comment:
Just checked and I don't see any rendering difference. As I said I find `code-block` to be more visible and explicit than `::` and since this is listing the way to run the commands the *console* highlight seems appropriate.